public/app/features/dashboard/state/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { PanelModel } from './PanelModel';
+
+export function isOnTheSameGridRow(sourcePanel: PanelModel, otherPanel: PanelModel): boolean {
+  if (
+    otherPanel.gridPos.x >= sourcePanel.gridPos.x + sourcePanel.gridPos.w &&
+    otherPanel.gridPos.y === sourcePanel.gridPos.y
+  ) {
+    return true;
+  }
+
+  return false;
```

Comment:
```suggestion
  return (
    otherPanel.gridPos.x >= sourcePanel.gridPos.x + sourcePanel.gridPos.w &&
    otherPanel.gridPos.y === sourcePanel.gridPos.y
  );
```
This code could be simplified a bit.

Reply:
Hmm, yes and no. IMO my version is more readable.